app.py

```diff
@@ -215,29 +254,14 @@ def save_identity():
     return jsonify({})
 
 
-# End Point example which verifies payload signature
-@app.route('/v1/signed_payload', methods=['POST'])
+# End Point example which recovers public address from signed payload
+@app.route('/v1/test_signed_payload', methods=['POST'])
 @validate_json
```

Comment:
Why do we need `@validate_json` decorator for `test_signed_payload`?

Reply:
no need, removed